serve/tests/unittest/test_engine_with_samplers.py

```diff
@@ -337,6 +339,43 @@ def _test_penalty(
     if use_staging_engine:
         engine.stop()
 
+def _test_logprobs(
```

Comment:
@vvchernov @zxybazh This test is broken, please fix.

Reply:
Hello @zxybazh, could you recheck it?

Reply:
Yep, I think it's broken, I'm working on a fix.

Reply:
Fixed in #183 Please check.